api/codes.py

```diff
@@ -17,15 +22,18 @@ def convert_country_code(city, country):
 
 def convert_city(city, country):
     city_index = list(filter(lambda x: city_data[x]['name_translations']['en'] ==
-                      city and city_data[x]['country_code'] == country, range(len(city_data))))
-    city_code = city_data[city_index[0]]['code']
+                      city and city_data[x]['country_code'] == country, range(len(city_data))))[0]
+    city_code = city_data[city_index]['code']
 
     # Check if the city exists
 
     return convert_airport_code(city_code)
 
 
-def convert_airport_code(city):
+def convert_airport_code(city) -> AirportData:
     airport_code = [{"name": x['name_translations']['en'], "code": x['code']}
                     for x in airport_data if x['city_code'] == city]
     return airport_code
+
+
+convert_country_code('Melbourne', 'Australia')
```

Comment:
Remove this. If you wanna do adhoc testing on functions, create a `testing.py` file, add it to `.gitignore`, then import the functions and run it there instead